src/hotspot/cpu/x86/stubGenerator_x86_64_arraycopy.cpp

```diff
@@ -33,6 +33,9 @@
 #ifdef COMPILER2
 #include "opto/c2_globals.hpp"
 #endif
+#if INCLUDE_JVMCI
+#include "jvmci/jvmci_globals.hpp"
+#endif
```

Comment:
Shouldn't that be "utilities/macros.hpp" to pick up the COMPILER2_OR_JVMCI macro?

Reply:
> Shouldn't that be "utilities/macros.hpp" to pick up the COMPILER2_OR_JVMCI macro?

Thanks @dean-long for the review.

You mean we should fix it like this?
```
diff --git a/src/hotspot/cpu/x86/stubGenerator_x86_64_arraycopy.cpp b/src/hotspot/cpu/x86/stubGenerator_x86_64_arraycopy.cpp
index b08e1f7..1d149c0 100644
--- a/src/hotspot/cpu/x86/stubGenerator_x86_64_arraycopy.cpp
+++ b/src/hotspot/cpu/x86/stubGenerator_x86_64_arraycopy.cpp
@@ -30,6 +30,7 @@
 #include "runtime/sharedRuntime.hpp"
 #include "runtime/stubRoutines.hpp"
 #include "stubGenerator_x86_64.hpp"
+#include "utilities/macros.hpp"
 #ifdef COMPILER2
 #include "opto/c2_globals.hpp"
 #endif
```

But it won't fix it actually.

Reply:
OK.  Sorry, I misunderstood the problem.